src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideRouter(routes)]
```

Comment:
suggestion

            providers: [provideRouter(routes) // Provides routing configuration]

          Explantion: Adding comments to this line clarifies what the provider is doing, enhancing readability.